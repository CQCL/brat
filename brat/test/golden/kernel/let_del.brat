ext "" X :: { Qubit -o Qubit }

f :: { Qubit -o Bool }
f = { q => let q' = X(q) in true }
