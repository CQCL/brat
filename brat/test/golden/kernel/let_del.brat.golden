Error in test/golden/kernel/let_del.brat on line 4:
f = { q => let q' = X(q) in true }
           ^^^^^^^^^^^^^^^^^^^^^

  Type error: Variable(s) q' haven't been used

